extensions/ql-vscode/src/vscode-tests/ensureCli.ts

```diff
@@ -44,7 +44,7 @@ const _10MB = _1MB * 10;
 
 // CLI version to test. Hard code the latest as default. And be sure
 // to update the env if it is not otherwise set.
-const CLI_VERSION = process.env.CLI_VERSION || 'v2.5.5';
+const CLI_VERSION = process.env.CLI_VERSION || 'v2.5.8';
```

Comment:
(Woops, I didn't know about this one! I haven't updated in the last few release 🤦🏽 )

Reply:
I forgot about that too. 🤷🏼 

Maybe we can change this to `latest` and never need to update it again.

Reply:
I think this shouldn't matter too much as it is only relevant when tests are run locally - the Actions workflow all set the `CLI_VERSION` environment variable that overrides this. Note that setting this to `nightly` (I assume that's what you meant since I don't think `latest` is a valid value?) won't work since that requires the `NIGHTLY_URL` environment variable to be set which defeats the point of this line (being able to run the tests locally where the environment variables _aren't_ set).

Reply:
Ah...I didn't know that the `NIGHTLY_URL` variable is required, so forget about the idea.  And, yes, this is low priority so if it is missed during an update, not a big deal.